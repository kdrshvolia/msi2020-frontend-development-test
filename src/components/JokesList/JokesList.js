import React from 'react';import PropTypes from 'prop-types';import JokeContainer from '../JokeContainer/JokeContainer';const JokesList = ({ jokesList }) => {  return (    <div>      {jokesList.map((joke) => (        <JokeContainer joke={joke} />      ))}    </div>  );};export default JokesList;