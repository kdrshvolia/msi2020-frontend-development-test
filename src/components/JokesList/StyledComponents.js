import styled, { css } from 'styled-components';const List = styled.div`  display: flex;  justify-content: space-between;  flex-direction: column;  align-items: center;  `;export { List };