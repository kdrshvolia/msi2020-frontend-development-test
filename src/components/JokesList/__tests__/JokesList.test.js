import React from 'react';
import renderer from 'react-test-renderer';
import JokesList from '../JokesList';

describe('SvgIcon', () => {
  it('renders correctly', () => {
    const toggleFav = jest.fn();
    const favJokes = {};
    const jokes = [];
    const rendered = renderer.create(
      <JokesList
        toggleFav={toggleFav}
        favoritedJokes={favJokes}
        jokesList={jokes}
        isFavoritedList
      />,
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
