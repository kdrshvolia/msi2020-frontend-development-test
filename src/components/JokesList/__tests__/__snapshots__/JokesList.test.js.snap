// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgIcon renders correctly 1`] = `
<div
  className="sc-fzozJi gPFRis"
/>
`;
